Utilities/Dockers/Alphashape/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:20.04
```

Comment:
could this just be `FROM python:something` and then all the python installation wouldn't be needed?